posts/2023-08-08_study_day/study_day.qmd

```diff
@@ -0,0 +1,84 @@
+---
+title: "It's all relative? - Calculating Relative Days using admiral"
+author:
+  - name: Ben Straub
+description: ""
+date: "2023-08-08"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [admiral, ADaMs]
+# feel free to change the image
+image: "admiral.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2023-08-08_study_day"
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here -----------------> 
+
+Creating `--DY` variables for your ADaMs is super easy using  [`derive_vars_dy()`](https://pharmaverse.github.io/admiral/v0.11.1/reference/derive_vars_dy.html) from the [admiral package](https://pharmaverse.github.io/admiral/cran-release/).
+
+
+Let's build some dummy data with 4 subjects, a start date/time for treatment (`TRTSDTM`), an analysis start date/time variable (`ASTDTM`) and an analysis end date variable (`AENDT`).
+
+```{r, message = FALSE}
+library(admiral)
+library(lubridate)
+library(dplyr)
+
+adam <- tribble(
+  ~USUBJID, ~TRTSDTM, ~ASTDTM, ~AENDT,
```

Comment:
I think we should align the commas of the columns.

Reply:
styler forced this non-alignment. Just going to leave as is.